db/migrate/20171112134503_add_available_to_games.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvailableToGames < ActiveRecord::Migration[5.0]
+  def change
+    add_column :games, :available, :integer
+  end
+end
```

Comment:
You don't need to add a column for this. Because we can tell if a game is available based on the `white_player_id` and `black_player_id`